tests/acceptance/test-helper.ts

```diff
@@ -26,3 +35,38 @@ export interface AppWithClient {
   app: PocketGatewayApplication
   client: Client
 }
+
+const localEnv = {
```

Comment:
Another nitpick, but can we move this to the top of the file and name it something like `DUMMY_LOCAL_ENV`? We're referencing this before it was defined, and it's better form in TS to have the "constant" declarations at the top of the file :).